client/login/magic-login/handle-emailed-link-form.jsx

```diff
@@ -110,7 +110,7 @@ class HandleEmailedLinkForm extends React.Component {
 			<EmptyContent
 				action={ action }
 				className={ classNames( {
-					'magic-login__isfetchingauth': isFetching,
+					'magic-login__is-fetching-auth': isFetching,
```

Comment:
Calypso uses a slightly tweaked version of [BEM](http://getbem.com/), and I would say keeping the prefix is best. Tentatively double-checking with @MichaelArestad.